worker/scripts/configure.py

```diff
@@ -85,6 +85,11 @@ def run_gyp(args):
     args.append('-Dclang=%d' % int(is_clang))
     if is_clang == False and major == 4 and minor <= 8:
       raise RuntimeError('gcc <= 4.8 not supported, please upgrade your gcc')
+  else:
+    if 'ninja' not in args:
+      args.extend(['-Goutput_dir=' + output_dir])
+      args.extend(['--generator-output', output_dir])
+
```

Comment:
Avoid two empty lines together.

Reply:
this :)

Reply:
Already removed these two lines for two reasons.
*  To generate Visual Studio solution file (.sln) and projects files (.vcxproj) just under `worker` directory. Then we can run `mediasoup-worker-test.exe` without changing the directory hierarchy of test resource files.
* The parameter `-Goutput_dir=` doesn't work, added settings into ` mediasoup-worker.gyp` 
```
# Output Directory setting for msvc. 
# https://github.com/nodejs/node-gyp/issues/1242#issuecomment-310921441
'msvs_configuration_attributes': {
     'OutputDirectory': '$(SolutionDir)\\out\\$(Configuration)\\'
 }
```